app/pathogen/arbovirus/analyze/recharts/estimate-count-by-un-region-and-arbovirus-graph.tsx

```diff
@@ -0,0 +1,103 @@
+import { useContext } from "react";
+import {
+  LegendConfiguration,
+  SlantedTick,
+  arbovirusesSF,
+  convertArboSFtoArbo,
+} from "../recharts";
+import { ArboContext } from "@/contexts/arbo-context";
+import {
+  typedGroupBy,
+  typedObjectEntries,
+  typedObjectFromEntries,
+} from "@/lib/utils";
+import { UNRegion, getLabelForUNRegion } from "@/lib/un-regions";
+import {
+  Bar,
+  BarChart,
+  CartesianGrid,
+  Legend,
+  ResponsiveContainer,
+  Tooltip,
+  XAxis,
+  YAxis,
+} from "recharts";
+import { pathogenColors } from "../../dashboard/(map)/ArbovirusMap";
+
+interface WhoRegionAndArbovirusBarInput {
+  legendConfiguration: LegendConfiguration;
+}
+
+export function EstimateCountByUnRegionAndArbovirusGraph(input: WhoRegionAndArbovirusBarInput) {
+  const state = useContext(ArboContext);
+  const legendProps =
+    input.legendConfiguration === LegendConfiguration.RIGHT_ALIGNED
+      ? {
+          layout: "vertical" as const,
+          verticalAlign: "middle" as const,
+          align: "right" as const,
+          wrapperStyle: { right: -10 },
+        }
+      : {
+          layout: "horizontal" as const,
+          verticalAlign: "bottom" as const,
+          align: "center" as const,
+          wrapperStyle: {
+            paddingTop: 10,
+            bottom: 0
+          }
+        };
+  
+  const dataToUse = state.filteredData.filter((dataPoint) => !!dataPoint.unRegion);
+  
+  const rechartsData = typedObjectEntries(
```

Comment:
This is different from the WHO region one but accomplishes the same thing. The casts would be unecessary if we were typegenning, which again is on my TODO list

Some kind of generalized graphing help is also on my TODO list, I know the copy-paste isn't ideal hahaha